shared/types/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@forgerock/shared-types",
+  "name": "@shared/types",
```

Comment:
hmm was this me or you? I think we should not do `@` in the name of a packge.json because to `npm` that infers its a scoped package (while this isn't an npm package, i just think it may be a good idea). Thoughts?

Reply:
Yeah, I'm okay with using a different symbol, or were you wanting no symbol at all? I kind of like the symbols as it makes it easier to know from scanning when something is a "special module" versus just a file within system.

Reply:
lets add this to our discussion